app/src/main/res/layout/schedule_event_list_item.xml

```diff
@@ -4,114 +4,114 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Can you delete this file and use `event_tile.xml` instead? I had to create an xml for the homepage tiles, so we should just use the same one across the app. You'll have to change `EventsAdapter.kt` to use the event_tile.xml and then update the view ids in that class.